src/index.tsx

```diff
@@ -254,7 +254,7 @@ export default class ReactToPrint extends React.Component<IReactToPrintProps> {
                     const node = canvasEls[i];
                     const contentDrawImage = node.getContext("2d");
                     if (contentDrawImage) {
-                        contentDrawImage.drawImage(srcCanvasEls[i] as HTMLCanvasElement, 0, 0);
+                        contentDrawImage.drawImage(srcCanvasEls[i], 0, 0);
```

Comment:
Was there a linting issue here?

Reply:
Yup, the `as` cast was unnecessary. The element was already that type.